modules/integration/src/test/java/io/ballerina/messaging/broker/integration/standalone/cli/InitCmdTest.java

```diff
@@ -118,4 +119,26 @@ public void testInitCmdWithParam() {
         Assert.assertEquals(configuration.getPassword(), "admin123", "password is invalid in the configuration file");
     }
 
+
+    @Test(description = "test configuration 'without password' invoked without providing a command password")
+    public void testNoPasswordParam() {
+        System.setProperty(TestConstants.CLI_CONFIG_SYSTEM_PROPERTY, InitCmdTest.TEMP_CONFIG_DIR + "cli-config3.yaml");
+
+        String[] cmd = { CLI_ROOT_COMMAND, "init" };
+        String expectedExpMessage = "User password is not provided.";
+
+        boolean doesContain = false;
+
+        Main.main(cmd);
+        // haven't used testNG builtin exception support because BrokerClientException doesn't support
+        // getDetailedMessage method
+        try {
+            Utils.getConfiguration(null);
+        } catch (BrokerClientException ex) {
+            doesContain = ex.getMessages().stream().anyMatch(msg -> msg.contains(expectedExpMessage));
+        }
+
+        Assert.assertTrue(doesContain);
```

Comment:
Shall we move this assert inside the catch block and add an assertion message?

Reply:
+1, forget to add the message. 
Asset inside catch block will not be okay right if the code doesn't throw an exception test will pass, but the feature will be broken.